src/Sarif.Sarifer/BackgroundAnalysisService.cs

```diff
@@ -46,11 +56,14 @@ public async Task AnalyzeAsync(string path, string text, CancellationToken cance
             {
                 DisposeStreams(streams);
             }
+
+            AnalysisCompleted?.Invoke(this, null);
```

Comment:
>AnalysisCompleted?.Invoke [](start = 12, length = 25)

Don't you want to do this in the `finally`? Otherwise you could end up with the menu items disabled forever.